src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -1686,6 +1686,7 @@ public static class UI
          public static readonly string DefaultSolubilityName = "Solubility";
          public static readonly string DefaultPopulationAnalysisType = "Population analysis";
          public static readonly string PreferredViewLayout = "Preferred View Layout";
+         public static readonly string TemplateReferenceBehavior = "Load metabolite when loading compound";
```

Comment:
"Load metabolites... " (plural) is probably better.

Reply:
done